src/pages/LogOutPreviousUserPage.js

```diff
@@ -0,0 +1,57 @@
+import React, {Component} from 'react';
+import lodashGet from 'lodash/get';
+import PropTypes from 'prop-types';
+import {withOnyx} from 'react-native-onyx';
+import ONYXKEYS from '../ONYXKEYS';
+import * as Session from '../libs/actions/Session';
+import FullScreenLoadingIndicator from '../components/FullscreenLoadingIndicator';
+import Navigation from '../libs/Navigation/Navigation';
+
+const propTypes = {
+    /** The parameters needed to authenticate with a short lived token are in the URL */
+    route: PropTypes.shape({
+        /** Each parameter passed via the URL */
+        params: PropTypes.shape({
+            /** The email of the transitioning user */
+            email: PropTypes.string,
+        }),
+    }),
+
+    /** The data about the current session which will be set once the user is authenticated and we return to this component as an AuthScreen */
+    session: PropTypes.shape({
+        /** The user's email for the current session */
+        email: PropTypes.string,
+    }),
+};
+
+const defaultProps = {
+    route: null,
+    session: null,
```

Comment:
**thought (blocker):** I want to say these are both `isRequired` props?

`route` since the screen is mounting in a navigator
`session` because a use is definitely logged in at this point

Reply:
Good point 👍